src/scenario.jl

```diff
@@ -565,7 +565,7 @@ function run_model(domain::Domain, param_set::NamedDimsArray, corals::DataFrame,
         p.X_mb .= tmp .* p.mb    # current cover * background mortality
 
         sol::ODESolution = solve(growth, solver, save_everystep=false, save_start=false,
-            alg_hints=[:nonstiff], abstol=1e-6, reltol=1e-6)  # , adaptive=false, dt=1.0
+            alg_hints=[:nonstiff], abstol=1e-6, reltol=1e-6,dt=0.5)  # , adaptive=false, dt=1.0
```

Comment:
Remove the commented code to avoid future confusion

```suggestion
            alg_hints=[:nonstiff], abstol=1e-6, reltol=1e-6,dt=0.5)
```

Reply:
Thanks, just changed. Also sorry, I just realised I forgot to add you as reviewer